EnRonSwanson/BangazonCLI
# Mb product

## Fixes - tests refactored

## Changes proposed in this pull request: 
Minor test refactoring in...
- OrderManagerShould
- ProductShould
- ProductManagerShould
- one line edit on Customer that @therealrmac said to add into this PR
...mostly to do with DateTime passing issues for Product

## How to test: 
- delete any instance of .db
- dotnet run- add a Customer and Product
- ensure adding to DB
- move to test directory and dotnet test
- 14/14 tests should pass

Please note that OrderManagerShould's Dispose method needs investigation. @arock83 is currently writing the SQL to go with it. Once it is written the Dispose will work and IDispose should be reenabled next PR.

@EnRonSwanson/
